bb/lib/cljfmt.clj

```diff
@@ -11,8 +11,9 @@
   "Runs cljfmt check on all files (recursively) in dir. Returns a collection of
   any files that failed the check."
   [dir]
-  (let [cljfmt-opts {:paths [dir], :diff? false}
-        {:keys [counts incorrect error] :as _result} (fmt/check cljfmt-opts)]
+  (let [cljfmt-opts {:diff? false}
```

Comment:
this isn't involved in this pr, but is this option to display a diff in the output? if that's the case, that seems helpful.

Reply:
It is, but I think in practice it's much more noise than signal. It still outputs messages saying where the problem is and a description of the specific problem, and that's always been more than sufficient to know what I need to do to fix it.

Here's an example:
```
clj-kondo lint:
src/fluree/server/consensus.clj:12:19-25::warning - unused binding thingy
```